client/src/js-sdk/sdk.js

```diff
@@ -1,49 +1,271 @@
 import db, { FIREBASE_FUNCTION_BASE_URL } from './fire.js';
+import firebase from 'firebase'
 import axios from 'axios';
+import uuidv4 from 'uuid/v4';
 
 // Listeners
-export function listenToZone(player, zone, callback) {
-  db.doc(`Games/game1/Players/${player}/Zones/${zone}`)
+export function listenToZone(gameId, player, zone, callback) {
+  db.doc(`Games/${gameId}/Players/${player}/Zones/${zone}`)
     .onSnapshot((doc) => {
       callback(doc.data())
     });
 }
 
-export function listenToPlayer(player, callback) {
-  db.doc(`Games/game1/Players/${player}`)
+export function listenToPlayer(gameId, player, callback) {
+  db.doc(`Games/${gameId}/Players/${player}`)
     .onSnapshot((doc) => {
       callback(doc.data())
     });
 }
 
 // Game functions
-export function tap(card) {
+// Card functions
+export function moveCardToZone(gameId, card, targetZone) {
+  if (card["state.zone"] === targetZone) {
```

Comment:
I'm not sure if I was unclear in the design doc, but `state` is an object with a property named `zone`. It won't be referenceable using `card["state.zone"]`. You could do either `card.state.zone` or `card["state"]["zone"]`.